examples/tests/client-tests.js

```diff
@@ -0,0 +1,182 @@
+/* global QUnit */
+
+'use strict';
+
+const BASE_MAIN_CHILDREN_NUMBER = 3;
+
+QUnit.module('Pictures');
+
+QUnit.test(
+  'putPicturesInPage',
+  (assert) => {
+    assert.ok(
+      typeof window.forTesting.putPicturesInPage === 'function',
+      'Create a function `putPicturesInPage` receiving an array of pictures. The function will add each picture and its attributes to a different section with the class `picture`.',
+    );
+
+    const elM = document.querySelector('main');
+    assert.strictEqual(
+      elM.childElementCount,
+      (0+BASE_MAIN_CHILDREN_NUMBER),
+      'Before using the function, the element `main` must be empty of pictures.',
+    );
+
+    // with an empty array
+    window.forTesting.putPicturesInPage([]);
+    const el = document.querySelectorAll('main section');
```

Comment:
this probably should only check for selector `main section.picture` because if we change the initial content of `main`, we'd break this test even though we're not breaking the putPicturesInPage function. This means rewriting the text below on line 31.